README.md

```diff
@@ -1176,6 +1197,20 @@ interface TTFBAttribution {
 }
 ```
 
+### Rating Thresholds:
```

Comment:
Can you move this section above the "Attribution:" section and below the "Functions:" section? Since this is part of the standard build, I think it makes sense to put it before the attribution stuff and keep that section last.

Reply:
[Done](https://github.com/GoogleChrome/web-vitals/pull/323/commits/daf6bed4793a46692816165353caab2debc6af2b)! Also [added a note](https://github.com/GoogleChrome/web-vitals/pull/323/commits/daf6bed4793a46692816165353caab2debc6af2b#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R984) encouraging the use of the `rating` supplied by the report callback functions:

> _**Note:** It's typically not necessary (or recommended) to manually calculate metric value ratings using these thresholds. Use the [`Metric['rating']`](#metric) supplied by the [`ReportCallback`](#reportcallback) functions instead._

Wdyt?

Reply:
Sure, I think that's helpful. Thanks!